src/index.html

```diff
@@ -1,14 +1,65 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <header class="header">
+      <a
+        href="https://www.moyo.ua/"
+        class="home-logo"
+        rel="noopener"
+        target="_blank"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo_logo"
+          class="logo-img"
+        >
+      </a>
+
+      <nav class="nav-block">
+        <ul class="menu-list">
+          <li>
+            <a href="#" class="menu-item apple is-active">Apple</a>
```

Comment:
don't use class names like apple or samsung if you change the text inside your item it will be weird